snap/local/cleanup.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
why not reusing https://github.com/ubuntu-robotics/snapcraft-ros-clean directly?